src/lib.rs

```diff
@@ -130,6 +147,17 @@ struct Inner {
 }
 
 impl Executor {
+
+    fn max_threads() -> usize {
+        match env::var(MAX_THREADS_ENV) {
+            Ok(v) => v.parse::<usize>().map(|v| {
+                if v < MIN_MAX_THREADS { MIN_MAX_THREADS } else { v }
+            }).map(|v| {
+                if v > MAX_MAX_THREADS { MAX_MAX_THREADS } else { v }
```

Comment:
```suggestion
                v.max(MIN_MAX_THREADS).min(MAX_MAX_THREADS)
```
I think you can use [`max`](https://doc.rust-lang.org/nightly/std/cmp/trait.Ord.html#method.max) and `min`.

Reply:
fixed.